src/Microsoft.Diagnostics.Monitoring.WebApi/IInProcessFeatures.cs

```diff
@@ -1,22 +1,23 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-
 namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
     public interface IInProcessFeatures
     {
-        bool IsCallStacksEnabled { get; }
-
-        bool IsExceptionsEnabled { get; }
-
-        bool IsParameterCapturingEnabled { get; }
-
+        /* Requirements */
         bool IsProfilerRequired { get; }
 
         bool IsStartupHookRequired { get; }
 
         bool IsHostingStartupRequired { get; }
 
         bool IsLibrarySharingRequired { get; }
+
+        /* Features */
```

Comment:
Why are these added back? These were explicitly removed in favor of the `IOptions<CallStackOptions>` and `IOptions<ExceptionsOptions>` services.